src/pages/VenuePage/VenuePage.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect, useCallback } from "react";
 import { Redirect, useHistory } from "react-router-dom";
+import firebase from "firebase/app";
```

Comment:
Can we just use the `useFirebase` hook instead of this?

Reply:
The objects are different, probably because the hook is being imported from `react-redux-firebase`. I checked the documentation of the library but there was nothing about calling functions.

Reply:
Based on your comment, I'm guessing that this was being used for a https function call or similar? If so, fair enough, not something `useFirebase` would have handled to my knowledge, if so fine to leave it as is.

(I wasn't able to check deeper to validate this as the force-push broke the code references, and this line doesn't seem to exist in the new diff of this file at all anymore)